src/foundry/common/documents/table-result.d.mts

```diff
@@ -45,7 +62,7 @@ declare abstract class BaseTableResult<
   /**
    * @remarks
    * Migrations:
-   * - Numeric `type`s to their new string values
+   * - Numeric `type`s to their new (since v12) string values
```

Comment:
Can you move `(since v12)` to the end of the line, sandwiched between seems weird.